src/views/Samples/scss/index.scss

```diff
@@ -0,0 +1,115 @@
+@import "../../../globalStyles/scss/variables";
+
+.sample-page {
+  padding: 17px;
+  background-color: $alabaster;
+}
+
+.collection {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 34px;
+
+  &-wrapper {
+    display: flex;
+    align-items: center;
+
+    &__main {
+      display: flex;
+      align-items: center;
+
+      &-name {
+        color: $dull-orange;
+        font-size: 14px;
+      }
+
+      &-icon {
+        margin-right: 5px;
+
+        & > div {
+          display: flex;
+        }
+      }
+    }
+
+    &__grid {
+      margin-right: 32px;
+    }
+  }
+}
+
+.wrapper-img {
+  display: grid;
+  grid-template-columns: repeat(2, auto);
+  column-gap: 14px;
+  row-gap: 14px;
+  margin-bottom: 34px;
+
+  &-main {
+    position: relative;
+    width: 100%;
+    padding-top: 100%;
+    background-color: $white;
+    height: 200px;
+    border-radius: 5px;
+
+    &-inner {
+      border: 1px solid transparent;
+      display: flex;
+      flex-direction: column;
+      height: 100%;
+      padding: 7px 22px;
```

Comment:
Use the variable $spacers or `rem` units for padding, margin.

Reply:
Done